drivers/gles3/shader_gles3.cpp

```diff
@@ -564,6 +564,13 @@ bool ShaderGLES3::_load_from_cache(Version *p_version) {
 			specialization.id = glCreateProgram();
 			glProgramBinary(specialization.id, variant_format, variant_bytes.ptr(), variant_bytes.size());
 
+			GLint link_status = 0;
+			glGetProgramiv(specialization.id, GL_LINK_STATUS, &link_status);
+			if (link_status != GL_TRUE) {
+				ERR_PRINT("Failed to load cached shader.");
```

Comment:
Since it's a "normal" error that will happen anytime users use a different GPU (integrated vs discrete), maybe this should only be a warning (or even a `print_verbose`?).
```suggestion
				WARN_PRINT("Couldn't load cached shader, recompiling.");
```